skimage/transform/_warps.py

```diff
@@ -178,6 +199,9 @@ def rescale(image, scale, order=1, mode=None, cval=0, clip=True,
     preserve_range : bool, optional
         Whether to keep the original range of values. Otherwise, the input
         image is converted according to the conventions of `img_as_float`.
+    multichannel : bool, optional
+        By default, is set to True for 3D (2D+color) inputs, and False for
```

Comment:
Sorry, I meant to add this reformulated sentence in addition to the description. It's not clear what this argument is for as it is.